LibreNMS/Validations/Database.php

```diff
@@ -77,6 +87,30 @@ public function validate(Validator $validator)
         $this->checkSchema($validator);
     }
 
+    private function checkVersion(Validator $validator)
+    {
+        $version = DB::selectOne('SELECT VERSION() as version')->version;
+        $version = explode('-', $version);
+
+        if (isset($version[1]) && $version[1] == 'MariaDB') {
+            if (version_compare($version[0], self::MARIADB_MIN_VERSION, '<=')) {
+                $validator->warn(
+                    'MariaDB version ' . self::MARIADB_MIN_VERSION . ' is the minimum supported version as of ' .
+                    self::MARIADB_MIN_VERSION_DATE . '. We recommend you update MariaDB to a supported version ' .
+                    self::MARIADB_RECOMMENDED_VERSION . ' suggested). If you do not update MariaDB, you will sooner or later run into problems.'
```

Comment:
Different wording here?

Reply:
good idea to check this.
why not a bool 'old_version' and point database type, version, and what is needed to variables
and after version_checks only one resulting string :-)

```
if (old_version): {
    $database_type . 'Version is ' . $running_version . '. Minimum Version is ' . $min_version . '. we recommend you to update your Database Server to at least Version ' . $recommended_version
}
```

Reply:
If you do not update MariaDB, you will sooner or later run into problems. -> Failure to update will eventually cause issues.